### PR TITLE
blacklists abductors from cult and clock cult

### PR DESCRIPTION
**Pull request by Blute1**
self explanatory, we don't need alien abductors while there are cult abductors in maintenance
:cl: Spacedong
tweak: Blacklisted the abductors random event from blood cult and clock cult gamemodes.
/:cl:

### CODE REVIEW

File: hippiestation.dme [hunk 1]
```diff
@@ -735,10 +735,10 @@
 #include "code\game\objects\effects\decals\decal.dm"
 #include "code\game\objects\effects\decals\misc.dm"
 #include "code\game\objects\effects\decals\remains.dm"
-#include "code\game\objects\effects\decals\Cleanable\aliens.dm"
-#include "code\game\objects\effects\decals\Cleanable\humans.dm"
-#include "code\game\objects\effects\decals\Cleanable\misc.dm"
-#include "code\game\objects\effects\decals\Cleanable\robots.dm"
+#include "code\game\objects\effects\decals\cleanable\aliens.dm"
+#include "code\game\objects\effects\decals\cleanable\humans.dm"
```
Comment (ghost) [inline hippiestation.dme:746]: stop changing this

Comment (Blute1) [reply hippiestation.dme]: i literally have no choice, its just there, i dont know why, im not changing it to fuck with you lmao

Comment (Blute1) [reply hippiestation.dme]: it doesn't do anything anyways

**Comment by dreamcrab** [comment]
hells no, abductors are great with any round

File: hippiestation.dme (hunk 1 above)
Comment (ghost) [reply hippiestation.dme:746]: dream maker auto-does it, lmao

Comment (GuyonBroadway) [reply hippiestation.dme:746]: Am I crazy or did you not include the file?

Comment (Blute1) [reply hippiestation.dme]: i thought I did

**Comment by banthebantz** [comment]
no it's fun

**Comment by Dudeinyourbackyard** [comment]
eh they dont change much
-1

**Comment by McBawbaggings** [comment]
abductors are good

cult rounds shouldn't even go on for long in terms of antag victory

**Comment by WallaceShark** [comment]
A lone abductor agent isn't going to influence the round any more than the large mob of cultists. -1.

**Comment by 2Dollarbill** [comment]
Abductors are a mini-antag role, I think they're fine as they are. -1

**Comment by Chnkr** [comment]
Your PR needs to be updated to reflect the current state of the master branch, please update your PR and either (re)open it, or ask a maintainer to (re)open it.

**Pull request closed without merging**